docs/books/admin_guide/03-commands.md

```diff
@@ -892,7 +892,7 @@ Example 2 - Displaying the contents of multiple files to standard output:
 $ cat /etc/passwd /etc/group
 ```
 
-Example 3 - Displaying the contents of several files in the file `usersAndGroups.txt`:
+Example 3 - Overwrite the contents of multiple files into one file (using output redirection):
```

Comment:
Hey @jimcat8... You are correct that the wording of Example 3 is incorrect, however, your re-write also needs some work. "Overwrite" at the beginning of the example is misleading. Yes we are overwriting the final output with our redirect (`>`) but we aren't overwriting the two files we are using as examples (`/etc/passwd` and `/etc/group`) which is what your rewording implies. I'd suggest instead: "Combining the contents of multiple files into one file using output redirection:"